BoxContentSDK/BoxContentSDK/BOXContentSDKErrors.m

```diff
@@ -16,7 +16,7 @@
 NSString *const BOXAuthTokenRequestErrorInvalidRequest = @"invalid_request";
 
 NSString *const BOXAuthErrorUnauthorizedDevice = @"unauthorized_device";
-NSString *const BOXAuthErrorExceededDeviceLimit = @"exceeded_device_limit";
+NSString *const BOXAuthErrorExceededDeviceLimit = @"exceeded_device_limits";
```

Comment:
@thuyBox Just confirmed with the API team that the error code is indeed 400 and the error type "exceeded_device_limits".